src/error.rs

```diff
@@ -79,6 +79,10 @@ quick_error! {
             display("unable to parse glob pattern")
             cause(err)
         }
+        AssetNotResolved(path: PathBuf) {
+            description("unable to resolve asset source path")
```

Comment:
Resolve here is an internal detail that users should not be concerned with. 

Maybe reword it to say simply that a file was not found?